### DISCUSSION TITLE
Improve DX for ESLint users by handling `--fix` flag

### DISCUSSION THREAD

**Discussion by wojtekmaj** [biomejs/biome#2459] [Ideas]
When doing

```
biome lint . --fix
```

We're met with the following error:

```
Error: --fix is not expected in this context
```

I get it, I get it - read the docs. But it would be nice to get something like:

```
Error: --fix is not expected in this context. Did you mean:
* --apply 
* --apply-unsafe
```

or something similar.

**Comment by Conaclos** [comment]
Indeed, this could be a great diagnostic. Any PR is welcome!

**Comment by unvalley** [comment]
I think this will be resolved the next release. 
https://github.com/biomejs/biome/pull/2898

so I close this.
